src/app.scss

```
@@ -1,15 +1,31 @@
-.header {
-  height: 100px;
-  width: 100%;
-  background-color: rgb(192, 192, 139);
+.main {
+  display: flex;
 }
 
-.logo {
+.startButton {
+  background-image: url(./assets/trollface.jpg);
+  background-size: cover;
+  width: 123px;
   height: 100px;
+  margin-top: 250px;
+  margin-left: 300px;
+  margin-right: 100px;
```

A shorthand makes sense here:

```suggestion
  margin: 250px 300px 100px 0px;
```